content/en/cloud_cost_management/datadog_costs/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Datadog Costs
+kind: documentation
+is_beta: true
+private: true
+disable_toc: false
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+- link: "/cloud_cost_management/aws"
+  tag: "Documentation"
+  text: "Gain insights into your AWS bill"
+- link: "/cloud_cost_management/azure"
+  tag: "Documentation"
+  text: "Gain insights into your Azure bill"
+- link: "/cloud_cost_management/google_cloud"
+  tag: "Documentation"
+  text: "Gain insights into your Google Cloud bill"
+- link: "/cloud_cost_management/custom"
+  tag: "Documentation"
+  text: "Gain insights into your custom costs"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Daily Datadog costs in Cloud Cost Management are in private beta. To request access, complete the form.
+{{< /beta-callout >}}
+
+## Overview
+
+You can enable daily Datadog costs in [Cloud Cost Management][1], giving you visibility into daily Datadog spending across dashboards, notebooks, [cost monitors][2], and Cloud Cost Analytics, along with your additional cloud provider and [SaaS costs][3] for your entire organization. 
+
+## Permissions
+
+<div class="alert alert-danger">Only existing Cloud Cost Management customers within the parent organization can access daily Datadog costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a>.</div>
+
+[Datadog costs][4] are available for users with [Datadog Admin][5] permissions or custom [Billing Read (`billing_read`) and Usage Read (`usage_read`)][6] permissions on the [**Usage & Cost** page][7].
+
+## Visualize costs
+
+<div class="alert alert-info">Costs outside of <a href="https://app.datadoghq.com/billing/usage">Plan and Usage</a> are based on usage only and do not include historical credits and adjustments.</div>
+
+{{< img src="cloud_cost/datadog_costs/dashboard.png" alt="Datadog costs as an option for the Cloud Cost data source in a dashboard" style="width:100%;" >}}
```

Comment:
not sure what this image is showing me relative to the paragraphs before or after it.